crates/driver/src/tests/setup/solver.rs

```diff
@@ -108,6 +109,13 @@ impl Solver {
                     order::Kind::Liquidity => "liquidity",
                     order::Kind::Limit { .. } => "limit",
                 },
+                "feePolicies": match quote.order.kind {
```

Comment:
I think there is also code ☝️  that should be dependent on the fee handler - lines 65 and 80

Reply:
Oh, totally true! thank you!